shipshape.yml

```diff
@@ -177,6 +177,9 @@ checks:
         - import configuration
         - synchronize configuration
         - use PHP for google analytics tracking visibility
+  drupal-admin-user:
+    - name: '[DATABASE] Active user roles admin check'
+      severity: high
```

Comment:
As a general rule of thumb, we usually look to implement new checks with low severity. With a follow up commit to change it to high in a subsequent release.

Given this check is already present for configuration files I think this is okay to have as a failure.